src/test/java/net/sf/jabref/logic/exporter/ModsExportFormatTestFiles.java

```diff
@@ -65,7 +62,7 @@ public void setUp() throws Exception {
         charset = StandardCharsets.UTF_8;
         modsExportFormat = new ModsExportFormat();
         tempFile = testFolder.newFile();
-        testImporter = new BibtexImporter(ImportFormatPreferences.fromPreferences(Globals.prefs));
+        testImporter = new BibtexImporter(Globals.prefs.getImportFormatPreferences());
     }
 
     @Test
```

Comment:
I can not use  `Standard.Charsets.UTF-8` on this branch. It does not exist. My master does not have it, too.